python/tests/test_highlevel.py

```diff
@@ -3206,10 +3206,19 @@ def test_apis(self):
             assert t1.get_mrca(*pair) == t1.mrca(*pair)
             assert t1.get_tmrca(*pair) == t1.tmrca(*pair)
 
+    @pytest.mark.filterwarnings("ignore::FutureWarning")
```

Comment:
Whats this doing? Shouldn't the warnings be caught by `pytest.warns`?

Reply:
Changed

Reply:
I think we want to ignore the warnings when testing the functionality (which is what this does), and then check for the warnings in the next function using `pytest.warns`?